app/components/primer/alpha/form_control.rb

```diff
@@ -3,8 +3,39 @@
 module Primer
   module Alpha
     # Wraps an input (or arbitrary content) with a label above and a caption and validation message beneath.
+    #
     # NOTE: This `FormControl` component is designed for wrapping inputs that aren't supported by the Primer
     # forms framework.
+    #
+    # @accessibility
```

Comment:
Love this! Can we add this note (maybe 1-2 sentences) in the preview itself too, in case consumers don't read the code?

Reply:
Woops, forgot to do that! Will add asap.